### PR TITLE
Create a "your HCB best friend" slide

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by sampoder** [comment]
Screenshot 2024-12-17 at 4 05 15 PM

I'm not sure if the face snow thing is too creepy? But I like it! Shame I can't figure out how to style these images.

File: components/Wrapped/slides/BestFriend.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { useTimedValue } from "./Start";
+
+export default function BestFriend({ data, isPaused }: SlideProps) {
+  if(!data.individual.bestFriend) {
+    return <></>
+  }
+
+  const snowflake1 = document.createElement("img");
+  snowflake1.src = data.individual.bestFriend.profile_picture; // @ts-ignore
+  snowflake1.style.cssText = "border-radius: 999px";
```
Comment (sampoder) [inline components/Wrapped/slides/BestFriend.tsx:17]: this doesn't seem to work :/

**Comment by garyhtou** [comment]
haha lol that's funny. i think we should keep the snow!

**Review by YodaLightsabr — APPROVED** [review]
LGTM!

**Pull request merged**